src/test/java/org/jenkinsci/plugins/ConfigurableExtensionFilterTest.java

```diff
@@ -23,4 +24,14 @@ public void cannotSetNull() {
         DescriptorImpl impl = new DescriptorImpl();
         impl.setExclusions(null);
     }
+
+    @Test
+    public void testConfigRoundtrip() throws Exception {
+        DescriptorImpl impl = new DescriptorImpl();
+        impl.setExclusions(new Exclusion[0]);
+
+        j.configRoundtrip();
+
+        assertEquals(0, impl.getExclusions().length);
```

Comment:
This assertion does not fail without the change to `src/main`, so is unrelated to the issue the `src/main` change addresses.